### PR TITLE
Expose signMessage and verifyMessageSignature

### PR DESCRIPTION
**Pull request by peterjan**
# PULL REQUEST

## Overview

This exposes two new methods on MySky, introduced here https://github.com/SkynetLabs/skynet-mysky/pull/134
See that PR for a more detailed description.

## Example for Visual Changes
N/A

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [x] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

## Issues Closed
N/A

### CODE REVIEW

File: src/mysky/index.ts [hunk 1]
```diff
@@ -640,6 +640,30 @@ export class MySky {
...
+
+  /**
+   * verifySignedMessage verifies the given message and returns the original
+   * message without signature if verification succeeded
+   *
+   * @param signedMsg - the signed message that needs to be verified
+   * @param publicKey - the public key to verify against
+   * @returns original message, or null in case verification failed
+   */
+  async verifySignedMessage(signedMsg: Uint8Array, publicKey: Uint8Array | string): Promise<Uint8Array | null> {
```
Comment (m-cat) [inline src/mysky/index.ts:663]: Waiting on the design discussion in the MySky PR to finalize whether we're going with the method signature here.

Comment (m-cat) [reply src/mysky/index.ts:663]: This method signature still needs to be updated.

**Review by m-cat — CHANGES_REQUESTED** [review]
Method signature should be updated.

File: src/mysky/index.ts (hunk 1 above)
Comment (peterjan) [reply src/mysky/index.ts]: push failed it seems, apologies

File: src/mysky/index.ts [hunk 2]
```diff
@@ -640,6 +640,32 @@ export class MySky {
...
+   * public key and returns a boolean that indicates whether the verification
+   * succeeded.
+   *
+   * @param message - the original message that was signed
+   * @param signature - the signature
+   * @param publicKey - the public key
+   * @returns boolean that indicates whether the verification succeeded
+   */
+  async verifyMessageSignature(message: Uint8Array, signature: Uint8Array, publicKey: string): Promise<boolean> {
+    return await this.connector.connection.remoteHandle().call("verifySignedMessage", message, signature, publicKey);
```
Comment (m-cat) [inline src/mysky/index.ts:666]:
```suggestion
    return await this.connector.connection.remoteHandle().call("verifyMessageSignature", message, signature, publicKey);
```

**Review by m-cat — CHANGES_REQUESTED** [review]
req: Update API call.

**Review by m-cat — DISMISSED** [review]

**Review by ro-tex — APPROVED** [review]

**Review by ro-tex — CHANGES_REQUESTED** [review]
Bump the version. Otherwise a client that has the `4.0.19-beta` version already cached locally will not be able to use this code.

**Comment by peterjan** [comment]
> Bump the version. Otherwise a client that has the `4.0.19-beta` version already cached locally will not be able to use this code.

cc @m-cat I bumped the version here, not sure how we usually do this? I figured it would be a separate PR immediately following it? Not sure.

**Review by ro-tex — DISMISSED** [review]

**Review by ro-tex — APPROVED** [review]

**Review by m-cat — APPROVED** [review]

**Pull request merged**